.github/scripts/verify_helm_install.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+NAMESPACE="scality-object-storage"
+
+echo "Verifying Helm installation..."
+
+if ! helm status scality-cosi-driver -n $NAMESPACE; then
+  echo "Helm release scality-cosi-driver not found in namespace $NAMESPACE"
+  exit 1
+fi
+
+echo "Verifying COSI driver Pod status..."
```

Comment:
Display timeout value to know what is waiting.
Idk if the command print during the timeout

```suggestion
echo "Verifying COSI driver Pod status for 120s..."
```